internal/backend/db/dbgorm/ownership.go

```diff
@@ -207,7 +216,17 @@ type PermissiveOwnershipChecker struct {
 	z *zap.Logger
 }
 
-func (c *PermissiveOwnershipChecker) IsUserEntity(db *gorm.DB, uid string, ids ...sdktypes.UUID) error {
+func (c *PermissiveOwnershipChecker) EnsureUserAccessToEntitiesWithOwnership(
+	db *gorm.DB, uid string, ids ...sdktypes.UUID,
+) ([]scheme.Ownership, error) {
+	oo, err := EnsureUserAccessToEntities(db, uid, ids...)
+	if err != nil && errors.Is(err, sdkerrors.ErrUnauthorized) {
+		err = nil // ignore not authorized, but keep all other (e.g. NotFound) - see var
```

Comment:
Do we want to log here?

Reply:
there is nothing to log in permissive checker - we allow access to all users. 
On the other hand in UserChecker we are logging of coz